### DISCUSSION TITLE
For a simple use case, can I include this library in its binary form in my application ?

### DISCUSSION THREAD

**Discussion by nebular** [nwjs-ffmpeg-prebuilt/nwjs-ffmpeg-prebuilt#169] [Q&A]
I have been trying to decipher the library license but cannot really understand if I am permitted to just include the DLL or DYLIB alongside my application. Even though I understand I should consult a lawyer, I would be most grateful if somebody with experience on the matter could just give me a "yes" or "no" for my simple use case, that I guess is the use case for typical applications (that do not modify ffmpeg source code but just needs the binary library to be available)

- The application uses the unmodified library in its binary form to play videos, so it is "a work that uses the library". Then it should be possible.
- However, the license says "There are certain libraries you can combine with FFmpeg whose licenses are not
compatible with the GPL and/or the LGPL. If you wish to enable these libraries, even in circumstances that their license may be incompatible, pass `--enable-nonfree` to configure. This will cause the resulting binary to be unredistributable.". Is this the case of this library ?

So, should / Can I:
- Just include the dylib/dll with my application installer
- Include the free version, and a link to this github so the final users can download the library and place it in the Program Folder, substituting the free one.

Thanks so much in advance.

**Comment by zkrige** [comment]
Honestly, I'm not really sure. I think it depends on the codecs you intend playing

**Reply by nebular** [reply]
Potentially all of them - as the app allows the user to drag & drop arbitrary videos from the computer
